cmd/lotus-shed/bitfield.go

```diff
@@ -109,43 +81,14 @@ var bitFieldRunsCmd = &cli.Command{
 
 var bitFieldStatCmd = &cli.Command{
 	Name:        "stat",
+	Usage:       "Bitfield stats",
 	Description: "print bitfield stats",
-	Flags: []cli.Flag{
-		&cli.StringFlag{
-			Name:  "enc",
-			Value: "base64",
-			Usage: "specify input encoding to parse",
-		},
-	},
```

Comment:
I'm pretty sure this flag is needed

Reply:
(and in other places)

Reply:
super are already enc
use func    Unified use of the upper level enc

Reply:
Name:        "bitfield",
        Usage:       "Bitfield analyze tool",
        Description: "analyze bitfields",
        Flags: []cli.Flag{
                &cli.StringFlag{
                        Name:  "enc",
                        Value: "base64",
                        Usage: "specify input encoding to parse",
                },
        },